docs/release-notes.txt

```diff
@@ -8,6 +8,128 @@
  Version 0.13
 **************
 
+Version 0.13.8
+==============
+
+**Release Date:** November 17, 2020
+
+**New Features**
+
+-  Deployment: When using ``det-deploy aws``, you can now specify the 
+   subnet that the agents should be launched into (if you are using the 
+   simple deployment type). This allows users to explicitly pick a
+   subnet/availability zone that has GPU instances.
```

Comment:
Not sure how to actually use this feature -- the docs (`how-to/installation/aws.txt`) haven't been updated, we don't give example usage here, and the relationship between subnets and AZs isn't immediately clear to me.

Reply:
cc @armandmcqueen

Reply:
How about:

When using the 'simple' deployment type with 'det deploy aws', you can now use the '--agent-subnet-id' flag to specify which existing subnet agents should be launched in. As each subnet is associated with a single availability zone, this allows users to explicitly choose an availability zone that has GPU instances (there is no public information about which availability zone has GPUs so trial and error is the suggested approach). 

I'm on my phone and can't find backticks, but 'det-deploy aws' and '--agent-subnet-id' should be formatted as code.